bodo/io/iceberg/theta.py

```diff
@@ -1,19 +1,59 @@
+"""
```

Comment:
This include both functions I moved from `stream_iceberg_write` and functions from the connector reimplemented using PyIceberg